source/administration/upgrade.md

```diff
@@ -21,11 +21,7 @@ To start, select one of the following guides:
 1. Download the **appropriate next upgrade** of your Team Edition server and note any compatibility procedures
       1. Run `platform -version` to check the current version of your Mattermost server
       2. Determine the appropriate next upgrade for your server:
-          - Mattermost `v3.0.x` and above can upgrade directly to Mattermost `v3.4.x`
-              - Note: If public links are enabled, upgrading to `v3.4.x` will invalidate existing public links due to a security upgrade allowing admins to invalidate links by resetting a public link salt from the System Console.
```

Comment:
Public links become invalidated if you upgrade from v3.3 or earlier to v3.5 though